course_gather/models.py

```diff
@@ -48,7 +48,45 @@ class Meta:
         db_table = 'colleges'
 
 
-class Course(models.Model):
+class ShellCourse(models.Model):
+    '''
+    Shell object of courses that is the intermediary between
+    Courses and MTUCourses
+    TransferCourse <=> Shell <=> MTUCourse
+
+    In order for this to work, a shell is going to have to related to many
+    Courses and only One MTUCourse
+    Example:
+    Cyber Sec (Two entries)
+
+    There's only one cyber sec at MTU
+    MTM<================>MTM
+    '''
```

Comment:
There's some additional notes about this, the main point though is that there will be data loss when we join these two data sets. Namely the strange $COURSE_CODE 1XXX courses that are pulled in from places. Along with "Unassigned transfers"